contents/blog/using-vc-carefully.md

```diff
@@ -0,0 +1,117 @@
+---
+date: 2023-05-16
+title: VC is a tool, not a religion
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-hawkins
+featuredImage: ../images/blog/posthog-ceo-diary-blog.png
+featuredImageType: full
+category: CEO diaries
+tags:
+  - Marketing
+---
+
+We raised over $27,000,000 then realized we should be growing out of our own revenue instead.
+
+## Day 1 - we started bootstrapping, then changed our minds
+
+Fresh-faced, but not carefree - in the early days, when it was just my cofounder, Tim, and me, we used to work out of cafés and we were bootstrapped.
+
+A few months in, we had a discussion on what we wanted out of life. Pro tip - do this _before_ you start working with someone else. Luckily we were on the same page - we decided it was _more_ stressful to bootstrap than to raise VC.
+
+Why?
+
+* No money coming in meant we were having to be exceptionally careful about spending anything.
+* We'd spend years rebuilding to the same salaries that we'd just left behind, but with added downside risk of not finding product market fit.
+* A slower pace felt like it'd be boring. We'd find it more engaging to own a smaller slice of a bigger pie. We wanted to work on something we'd enjoy.
+
+The Y Combinator application was coming up, so we went for it. We got in.
+
+## Raising money got intoxicating
+
+We raised a seed round fairly painfully (March 2020...). Then a Series A super quickly (a month later), then a B super duper quickly (about 6 months later, announced way after it happened).
+
+Raising money, especially coming into 2021, felt self fulfilling. It felt like you're _either_ the type of company that raises a ton, or one that struggles. So we'd better keep raising. This was the hype loop we felt was happening:
+
+1. Founder: Check it out, we raised tons of money
+2. VC: Whoah, that's some compelling social proof! You're now on our radar.
+3. Users: We should pick _this_ technology since they're featured in techcrunch with a huge funding round!
+4. Founder: We should keep raising. If we can raise the next round right now, not doing that feels stupid.
+5. VC: Interest rates are low... let's do it.
+6. Go back to step 1 (until founder raises too much, has big unweildy team and no product market fit, with high burn, has to fire people and will struggle to raise again)
```

Comment:
```suggestion
1. **Founder:** "Check it out, we raised tons of money!"
2. **VC:** "Whoah, that's some compelling social proof! You're now on our radar."
3. **Users:** "We should pick _this_ technology since they're featured in TechCrunch with a huge funding round!"
4. **Founder:** "We should keep raising. If we can raise the next round right now, not doing that feels stupid."
5. **VC:** "Interest rates are low... let's do it."
6. **Go back to step 1** (until founder raises too much, has big unwieldy team and no product-market fit, with high burn, has to fire people and will struggle to raise again)
```